sklearn/model_selection/tests/test_validation.py

```diff
@@ -1628,6 +1628,100 @@ def test_permutation_test_score_pandas():
         permutation_test_score(clf, X_df, y_ser)
 
 
+def test_fit_and_score_verbose():
```

Comment:
maybe you'll want to make use of the `capsys` fixture https://docs.pytest.org/en/latest/capture.html

Reply:
Yes, totally. I just realized that the test I removed is more in line with recent recommendation. Sorry for that.
If this is ok, I will reintroduce the removed test and modify it for the new format, rather than keep the older one.

Reply:
that sounds good!
And no need to apologize, none of us get it right the first time!

Reply:
Fixed.